### DISCUSSION TITLE
Kubernetes CRDs for PlanetScale Databases

### DISCUSSION THREAD

**Discussion by austince** [planetscale/discussion#185] [Feature Requests] [answered]
Hey all, wondering if there has been any work to allow deploying PlanetScale DBs via k8s custom resources. Thinking either a custom operator, a [crossplane Provider](https://crossplane.io/docs/master/concepts/providers.html), etc.

**Comment by jgreet** [comment, marked as answer]
Hi, as of we now we don't have plans to become a Crossplane provider or publish an operator for hosted databases. 

We do provide a [Go client library](https://github.com/planetscale/planetscale-go/) that could be used if someone was interested in writing their own.
